conbench/util.py

```diff
@@ -13,7 +13,12 @@
 import yaml
 from _pytest.pathlib import import_path
 from requests.adapters import HTTPAdapter
-from requests.packages.urllib3.util.retry import Retry
+
+try:
+    from urllib3.util import Retry
+except ImportError:
+    # Legacy around times where requests had version 2.3 something.
+    from requests.packages.urllib3.util.retry import Retry  # type: ignore[no-redef]
```

Comment:
Oh boy.
https://github.com/python/mypy/issues/1153#issuecomment-942573356
(the entire thread)